gnpy/core/network.py

```diff
@@ -534,8 +531,7 @@ def build_network(network, equipment, pref_ch_db, pref_total_db):
     for fiber in fibers:
         split_fiber(network, fiber, bounds, target_length, equipment)
 
-    amplified_nodes = [n for n in network.nodes()
-                        if isinstance(n, Fiber) or isinstance(n, Roadm)]
+    amplified_nodes = [n for n in network.nodes() if isinstance(n, Fiber) or isinstance(n, Roadm)]
```

Comment:
This list is to identify nodes that will need to have a amplifier (egress) in case the user did not put one (autodesign): shouldn't there be a  `or isInstance(n, RamanFiber)` here ?

Reply:
Since RamanFiber inherit from Fiber, a RamanFiber instance would be also a Fiber instance